lib/rust/ensogl/component/scroll-area/src/lib.rs

```diff
@@ -16,14 +16,17 @@
 #![warn(unused_import_braces)]
 #![warn(unused_qualifications)]
 
+use ensogl_core::display::shape::*;
```

Comment:
I don't like the idea to import everything from shape here, because `scroll_area` is not logically connected to shape. Use `use ...::shape` instead, please.

Reply:
This is a pretty common pattern in our codebase (53 places) that is probably caused by the incorrect implementation of `define_shape_system` macro. It was not importing all the necessary traits into the scope. I fixed it and used qualified `shape::` imports in this module. Should I also change all other places where `use shape::*` is used? If so - please specify if I should prioritize qualified access in form of `shape::Rect` or shorter code lines. Also, I'm not sure whether my `use shape;` should be used in all places or only in modules that are "not logically connected to shape".